src/main/java/nl/novi/techiteasy/dtos/TelevisionMapper.java

```diff
@@ -0,0 +1,74 @@
+package nl.novi.techiteasy.dtos;
+
+import nl.novi.techiteasy.models.Television;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
```

Comment:
Hier is deze annotatie wel nuttig, omdat deze klasse via injection wordt gebruikt in de service.